### DISCUSSION TITLE
Blank Jupyter Notebook

### DISCUSSION THREAD

**Discussion by mccabete** [MAAP-Project/Community#784] [Platform] [answered]
When logging onto MAAP @ ade.maap-project.org, my notebooks appear "blank" (See image attached). I have tried refreshing the browser, stopping and re-starting the workspace, and starting a brand-new workspace. Nothing seems to work. Any suggestions for how to work with these notebooks? 

<img width="1470" alt="image" src="https://github.com/MAAP-Project/Community/assets/21266199/2196e1ed-1c56-4068-b144-e0ebde9353f2">

**Comment by sujen1412** [comment]
Are these notebooks created under any of the my-public/private-buckets folders ? 
If yes, then they will not work since its an S3 backed file system.

**Comment by mccabete** [comment]
@sujen1412 No, they are at a `/projects/sub_folder/` path.

**Comment by rtapella** [comment]
probably same as: https://github.com/MAAP-Project/Community/issues/735

**Comment by rtapella** [comment]
(well.. the bug is not the same but both issues are probably caused by the Jupyter collaboration feature)

**Reply by mccabete** [reply]
@rtapella I am still getting the blank notebook stuff, even though the collaboration fix went through AFAIK.

**Reply by mccabete** [reply]
Huh I seem to still be getting the cursor reset issue too. Is there something I need to do to get the changes to take effect?

**Reply by sujen1412** [reply]
@mccabete - Just to confirm, are you still using the workspaces you used before the release of v3.1.0 ? 
Or are you seeing this issue in the new v3.1.0 workspaces ?

**Reply by mccabete** [reply]
They are the same workspaces, yeah. I had not been able to make a new workspace, but that has since resolved. Things are working!

**Reply by sujen1412** [reply]
Great! Please close this ticket once you confirm its working as expected in the new workspace.

**Comment by rtapella** [comment]
PR for potential fix: https://github.com/MAAP-Project/maap-workspaces/pull/53

**Comment by mccabete** [comment]
I realize that there may be a fix in the works, but in the meantime, are there workarounds so I can use my notebooks?

**Reply by mccabete** [reply, marked as answer]
On [the slack](https://esa-nasa-maap.slack.com/archives/CLQB3B620/p1691430387218229?thread_ts=1691429258.867129&cid=CLQB3B620), Jamison French pointed out that I could duplicate the notebooks and render them.
